src/integTest/groovy/com/readytalk/LifecycleIntegTest.groovy

```diff
@@ -10,7 +10,7 @@ class LifecycleIntegTest extends IntegrationSpec {
       apply plugin: 'com.readytalk.ci'
       apply plugin: 'java'
 
-      buildEnv.isCI = false
+      buildEnv.ci = false
```

Comment:
I would rather struggle through the last Java 6 projects than try to maintain Java 6 compatibility.